src/dom.ts

```diff
@@ -48,7 +48,11 @@ export const registerDom = <ElementType extends TramOneHTMLElement | TramOneSVGE
 				// reset working key so we have the correct place when starting a new component
 				restoreWorkingKey(TRAM_HOOK_KEY, uniqueBranch);
 
-				return tagFunction(props, children);
+				const processedChildren: Children = children?.map((child: unknown | Element) => {
+					if (child instanceof Element) return child;
+					return `${child}`;
+				});
+				return tagFunction(props, processedChildren);
```

Comment:
previously we weren't stringifying children, and using template variables actually creates non-string children!

e.g. `<div>Hello ${() => {}}</div>` would previously generate a child element that is of type `function`.

Since other parts of the code (and Tram-One in general) expect either a `string` or an `Element`, we force this processing.